pkg/pillar/cmd/zedkube/zedkube.go

```diff
@@ -338,6 +452,61 @@ func Run(ps *pubsub.PubSub, loggerArg *logrus.Logger, logArg *base.LogObject, ar
 		ps.StillRunning(agentName, warningTime, errorTime)
 	}
 	log.Noticef("zedkube run: device network status initialized")
+
+	//
+	// NodeDrainRequest subscriber and NodeDrainStatus publisher
+	//
+	// Sub the request
+	subNodeDrainRequest, err := ps.NewSubscription(pubsub.SubscriptionOptions{
```

Comment:
probably should name this similar to BoM by adding the NA (node agaent) at the end to be clearer

Reply:
Sure, I can do that

Reply:
@naiming-zededa renamed to subNodeDrainRequestZA since they subscribe to agent zedagent.